Dockerfile

```diff
@@ -135,7 +135,7 @@ RUN git init . && git remote add origin "https://github.com/go-delve/delve.git"
 # from the https://github.com/go-delve/delve repository.
 # It can be used to run Docker with a possibility of
 # attaching debugger to it.
-ARG DELVE_VERSION=v1.9.1
+ARG DELVE_VERSION=v1.20.1
```

Comment:
Are they moving that fast, or did we just pick a very old version before? (because it was updated not "too long ago" https://github.com/moby/moby/pull/44713) 😂

Reply:
it jumped from 1.9 to 1.20.